spec/features/services/create_collection_association_spec.rb

```diff
@@ -0,0 +1,152 @@
+require 'rails_helper'
+
+describe 'Creating Service Collection Associations', js: true, reset_provider: true do
+  before do
+    login
+
+    @ingest_response, _concept_response = publish_service_draft
+  end
+
+  before :all do
+    # Make 3 collections, two with unique names so that we can ensure they
+    # are returned from search correctly
+    @ingest_response1, _concept_response1 = publish_collection_draft(entry_title: 'MODIS-I Water Traveler')
```

Comment:
All the ingest responses start with `@ingest_response` even though there are services and collections - maybe they should be more differentiated?